content/docs/user-guide/experiment-management/experiments-overview.md

```diff
@@ -0,0 +1,27 @@
+# DVC Experiments Overview
+
+DVC Experiments are captures automatically by DVC when you [run them].
+
+[run them]: /doc/user-guide/experiment-management/running-experiments
+
+## Basic Workflow
```

Comment:
Should we name the whole section as "Basic workflow" if that's what this section is about anyway

Reply:
> name the whole section as "Basic workflow" if that's what this section is about

There is another intention here actually, which comes before the workflow. From https://github.com/iterative/dvc.org/pull/2909#discussion_r740633633:

>2... document implementation details about experiments (Git custom refs, etc.).

I just didn't do it yet because I thought it's planned in https://github.com/iterative/dvc.org/issues/2768 and didn't want to further complicate this PR which again is starting to grow beyond the original scope in terms of file changes... Maybe I'll start a nested one just to show what it looks like ⌛

Reply:
> > document implementation details about experiments
>
> I'll start a nested one just to show what it looks like

@shcheklein @iesahin please see https://github.com/iterative/dvc.org/pull/3006/files for this purpose. If you approve both PRs I'll merge the nested one first and then this one.